assets/stylesheets/common/discourse-post-event.scss

```diff
@@ -205,7 +246,7 @@ $interested: #fb985d;
     flex-direction: column;
 
     .header {
-      display: flex;
+      display: none;
```

Comment:
why do we have it if we don't show it?

Reply:
I think I felt it was too drastic to remove for now? This is a hard one. If someone still wants this header with the "show all" button, I was afraid of removing it.

I am fine to remove it though as we are still giving people the option to show header in the ellipsis menu.